src/Usages/ClassConstantUsages.php

```diff
@@ -78,11 +80,11 @@ public function processNode(Node $node, Scope $scope): array
 		} else {
 			if ($usedOnType->hasConstant($constant)->no()) {
 				return [
-					sprintf(
+					RuleErrorBuilder::message(sprintf(
 						'Cannot access constant %s on %s',
 						$constant,
 						$usedOnType->describe(VerbosityLevel::getRecommendedLevelByType($usedOnType))
-					),
+					))->build(),
```

Comment:
Note to self: no need for `->identifier(...)` here because the error is not a *disallowed call* error.